### ISSUE TITLE
asking to "Close" Door or GARAGE device will "Lock" it.

### ISSUE THREAD

**Issue by mwinters-stuff** [actions-on-google/actions-on-google-nodejs#322]
Since friday, When asking to "Close Garage Door", the server receives "Lock Garage Door" command.
Open works fine.

So.. 
![Screenshot_20190514-173226_Google](https://user-images.githubusercontent.com/2851384/57673066-d1367d00-766e-11e9-8e14-9c7138be3432.png)

My devices, are configured as such (this is a test device on my bench)
```json
      {
        "id": "l4kp3v7T",
        "type": "action.devices.types.GARAGE",
        "traits": [
          "action.devices.traits.OpenClose",
          "action.devices.traits.LockUnlock",
          "action.devices.traits.Sensor"
        ],
        "name": {
          "defaultNames": ["Door"],
          "name": "Door",
          "nicknames": ["Office Door"]
        },
        "attributes": {
          "dataTypesSupported": [
            {
              "name": "temperature",
              "data_type": [{ "type_synonym": ["temperature"], "lang": "en" }],
              "default_device_unit": "C"
            }
          ]
        },
        "willReportState": true,
        "deviceInfo": {
          "manufacturer": "XXXX",
          "model": "joe",
          "swVersion": "1.0.0",
          "hwVersion": "1.0"
        }
      }

```

This is the request received for "Open"
```json
{
  "inputs": [
    {
      "context": { "locale_language": "en" },
      "intent": "action.devices.EXECUTE",
      "payload": {
        "commands": [
          {
            "devices": [{ "id": "l4kp3v7T" }],
            "execution": [
              {
                "command": "action.devices.commands.OpenClose",
                "params": { "openPercent": 100 }
              }
            ]
          }
        ]
      }
    }
  ],
  "requestId": "1216667600648946356"
}

```

This is the execute received when I ask to 'Close Door'

```json
{
  "inputs": [
    {
      "context": { "locale_language": "en" },
      "intent": "action.devices.EXECUTE",
      "payload": {
        "commands": [
          {
            "devices": [{ "id": "l4kp3v7T" }],
            "execution": [
              {
                "command": "action.devices.commands.LockUnlock",
                "params": { "lock": true }
              }
            ]
          }
        ]
      }
    }
  ],
  "requestId": "8348603030811304316"
}
```

Is anyone else having this problem?

**Comment by Fleker** [comment]
Does this still happen if the device does not have `LockUnlock`, just the `OpenClose` trait?

**Comment by wintersandroid** [comment]
I have not tried that, I can give it a go when I get home tonight. Still seems strange though, If I "Close Door" while the door is close it locks as well..

**Comment by wintersandroid** [comment]
I have tried commenting out the LockUnlock (and sensor) traits, so only OpenClose is reported and 
synced, Then the Close command works as expected.

Here is a log, where LockUnlock trait is removed, From startup of the server to me "Opening Office Door" then "Close Office Door" This worked good.
[OpenCloseNoLockUnLock.log](https://github.com/actions-on-google/actions-on-google-nodejs/files/3181132/OpenCloseNoLockUnLock.log)


Then done again, This time with LockUnlock trait enabled, Just had to issue "Close Office Door" even though the door was already reported closed. I got the execute command for lock.
[OpenCloseWithLockUnlockFail.log](https://github.com/actions-on-google/actions-on-google-nodejs/files/3181135/OpenCloseWithLockUnlockFail.log)
